garak/detectors/packagehallucination.py

```diff
@@ -143,3 +143,58 @@ def detect(self, attempt: Attempt) -> List[float]:
             scores.append(1.0 if hallucinated_gem else 0.0)
 
         return scores
+
+
+class JavaScriptNpm(Detector):
+    """Check if the output tries to import or require an npm package not listed in the npm registry"""
+
+    bcp47 = "en"
+
+    npm_packages = None
+    # TODO update this with garak-llm dataset
+    npm_dataset_name = "arjun-krishna1/npm-20240828"
```

Comment:
Moving this to garak-llm huggingface org once I'm given access

Reply:
enabled

Reply:
Thanks! Moved over dataset to garak-llm org and updated reference in PR